packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -396,6 +396,8 @@ export async function generateImageData({
     imageProps.placeholder = {
       fallback,
     }
+    // @ts-ignore - Type is incorrect because it's removed
+    // TODO(v6): Remove this codepath
```

Comment:
We can remove this already - we should not get into situation of ever hitting this code path anymore because earlier in function we swap `tracedSVG` to `dominantColor` in https://github.com/gatsbyjs/gatsby/blob/e761986e36878a6b8dea4f123771093328b78a90/packages/gatsby-plugin-sharp/src/image-data.ts#L149-L157 and this conditional should be dead already